pkg/helper/client.go

```diff
@@ -0,0 +1,77 @@
+package helper
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"os/exec"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	nmstatev1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1"
+)
+
+const nmstateCommand = "nmstatectl"
+const namespace = "default"
```

Comment:
Isn't NodeNetworkState cluster-wide? You don't need to specify Namespace name when handling cluster-wide resources.

Reply:
It's not working if I don't use namespace.

Reply:
Aha, we are not cluster-wide yet: https://github.com/nmstate/kubernetes-nmstate/blob/operators-wip/deploy/crds/nmstate_v1_nodenetworkstate_crd.yaml#L12. Nevermind then, we have to fix that in another PR

Reply:
Damn it next PR will be before starting with desiredState.